db/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM postgis/postgis:15-3.4-alpine
+FROM postgres:16-bookworm
```

Comment:
This looks like the right approach to me.

Reply:
On reflection, I think I should downgrade this back to 15-bookworm until we're ready to make a collective effort to upgrade to 16.